website/static/resources.html

```diff
@@ -48,7 +48,22 @@ <h1>Resources</h1>
 <div class="content homecontent width clearfix">
     <div class="leftcol widecol faq">
 
-        <h2 id="tools">Presto Management Tools</h2>
+<h2 id="tutorials">Tutorials</h2>
+        <div class="item">
+            <h4>PrestoDB Sandbox on AWS</h4>
+            <dl>
+                <dt>Website</dt>
+                <dd><a href="https://aws.amazon.com/marketplace/pp/B08C21CGF6?ref_=srh_res_product_title" target="_blank">PrestoDB Sandbox AMI</a></dd>
+                <dt>Maintained by</dt>
+                <dd><a href="https://ahana.io/" target="_blank">Ahana</a></dd>
+                <dt>Description</dt>
+                <dd>
+		 Learn free open source Presto. Sandbox comes bundled with catalogs to try, test, and prototype Presto 5 min tutorial 		on getting started at <a href="https://ahana.io/tutorials/aws-sandbox/" target="_blank">https://ahana.io/tutorials/aws-sandbox/</a>                
```

Comment:
@ahana-dev please fix the text here - 5 min tutorial seems out of place.

Reply:
Please review - text is now fixed @dborkar